helm/ingress-azure/templates/deployment.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         checksum/config: {{ print .Values | sha256sum }}
         prometheus.io/scrape: "true"
         prometheus.io/port: {{ .Values.kubernetes.httpServicePort | quote}}
+        {{- with .Values.podAnnotations }}
```

Comment:
can you add this field under `.Values.kubernetes`

Reply:
ok, annotations can be added from `.Values.kubernetes.podAnnotations` now